CHANGES/1394.feature

```diff
@@ -0,0 +1,2 @@
+Create POST {repo_href}/copy_collection_version/ and POST {repo_href}/copy_collection_version/
```

Comment:
```suggestion
Created ``POST {repo_href}/copy_collection_version/`` and ``POST {repo_href}/copy_collection_version/``
```
We use past tense. Also please keep in mind that these will (currently) be rendered with RST.

Did you intend to name one of them differently?